tests/e2e/Taskfile.yaml

```
@@ -28,3 +33,35 @@ tasks:
     cmds:
       - ginkgo
 
+  run_local:
+    desc: "Run localy e2e tests"
+    deps:
+      - virtctl
+      - ginkgo
+    cmds:
+      - |
+        export KUBECONFIG={{.KUBECONFIG}}
+        export E2E_CLUSTERTRANSPORT_KUBECONFIG={{.E2E_CLUSTERTRANSPORT_KUBECONFIG}}
+        ginkgo
+    vars:
+      KUBECONFIG: '{{default "$HOME/.kube/config" .KUBECONFIG}}'
+      E2E_CLUSTERTRANSPORT_KUBECONFIG: '{{default "$HOME/.kube/config" .E2E_CLUSTERTRANSPORT_KUBECONFIG}}'
+
+  run_one:
+    desc: "Run one test or group"
+    deps:
+      - virtctl
+      - ginkgo
+    cmds:
+      - |
+        {{if .TEST }}
+          export KUBECONFIG={{.KUBECONFIG}}
+          export E2E_CLUSTERTRANSPORT_KUBECONFIG={{.E2E_CLUSTERTRANSPORT_KUBECONFIG}}
+          ginkgo --focus "{{ .TEST }}" -v
+        {{else}}
+          echo "Test not passed"
+          echo 'Example: TEST="Label and Annotation" task run_one'
```

Shouldn't it fail if "not passed"?
```suggestion
          echo 'Example: TEST="Label and Annotation" task run_one'
          exit 1
```

---

exit 1 - not necessary 
`echo "Test not passed"` changed to `echo "Specify test to run"`